src/app/services/data.service.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class DataService {
```

Comment:
It's difficult to understand what this service does, it has too generous name, no encapsulation, just one public field that is bad practice